src/Symmetric/SymmetricInterface.php

```diff
@@ -55,7 +76,23 @@ public function getBlockSize();
      */
     public function setMode($mode);
 
+    /**
+     * @return string
+     */
     public function getMode();
 
+    /**
+     * @return array
+     */
     public function getSupportedModes();
+
+    /**
+     * @return array
+     */
+    public function getOptions();
```

Comment:
Same note here as in the implementation; is this a method with a use case? Particularly considering the various individual options each have getters already, it's largely redundant…